packages/hint-compat-api/src/css.ts

```diff
@@ -276,12 +277,22 @@ export default class CSSCompatHint implements IHint {
                 const location = getCSSLocationFromNode(node, { isValue });
                 const severity = alternatives.length ? Severity.error : Severity.warning;
 
+                let documentation: ProblemDocumentation[] | undefined;
+
+                if (unsupported.mdnUrl) {
+                    documentation = [{
+                        link: unsupported.mdnUrl,
+                        text: getMessage('learnMoreCSS', context.language)
+                    }];
+                }
```

Comment:
Minor nit to reduce explicit types and allow this to be `const`. Feel free to ignore if you don't like the format.

```suggestion
                const documentation = unsupported.mdnUrl && [{
                    link: unsupported.mdnUrl,
                    text: getMessage('learnMoreCSS', context.language)
                }];
```

Reply:
Looks good :)